ibazel/main.go

```diff
@@ -81,13 +84,17 @@ func parseArgs(in []string) (targets, bazelArgs, args []string) {
 
 // main entrypoint for IBazel.
 func main() {
-	if len(os.Args) < 3 {
+	flag.Usage = usage
+	flag.Parse()
+
+	if len(flag.Args()) < 2 {
 		usage()
 		return
 	}
 
-	command := strings.ToLower(os.Args[1])
-	args := os.Args[2:]
+	command := strings.ToLower(flag.Args()[0])
+	args := flag.Args()[1:]
+	fmt.Printf("Command: %s Args: %v", command, args)
```

Comment:
Is this print statement worth keeping around? In most use cases this information is present from what the user just typed. I think it'll look something like

ibazel run //path/to:target
Command: run Args: //path/to:target

ibazel run //path/to:target --bazelPath=/path/to/bazel
Command: run Args: //path/to:target --bazelPath=/path/to/bazel